internal/services/network/private_endpoint_resource.go

```diff
@@ -196,6 +195,13 @@ func resourcePrivateEndpoint() *pluginsdk.Resource {
 							ForceNew:     true,
 							ValidateFunc: validation.StringIsNotEmpty,
 						},
+						"member_name": { // TODO: set to Required, remove Computed in v4.0
```

Comment:
could we make this change now and put it behind the 4.0 flag instead of a todo please? thanks